### PR TITLE
Ygo 23

### PR DESCRIPTION
**Pull request by Erakedon**

### CODE REVIEW

File: src/Shared/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    ];
+
+    currentPath = "default";
+    previousPath = "";
+
+    renderPageName () {
+        const pageName = this.urlToPagenameCorelation.find((el) => {
+            return el.url === this.props.location.pathname
+        }).name;
+        console.log(this.props.location);
```
Comment (NightAngell) [inline src/Shared/Header/Header.js]: Remove log.

File: src/Shared/Nav/Nav.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import NavButton from './NavButton';
+
+class Nav extends Component {
+    state = { 
+        navCollapsed: true
+     }
+
+     toggleNavBar() {
+         console.log(this.props);
```
Comment (NightAngell) [inline src/Shared/Nav/Nav.js]: Remove logs.

File: src/Shared/Recharts/BasicLineRechart.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { LineChart, Line, CartesianGrid, XAxis, YAxis, Tooltip } from 'recharts';
+import Colors from '../../colors';
+
+const BasicLineRechart = (props) => {
+    console.log(Colors);
```
Comment (NightAngell) [inline src/Shared/Recharts/BasicLineRechart.js]: Remove log.

**Review by NightAngell — CHANGES_REQUESTED** [review]
Remove logs. Everything else look fine.

**Review by NightAngell — APPROVED** [review]

**Review by szefo09 — APPROVED** [review]

**Pull request merged**